core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -50,7 +50,8 @@ oppia.factory('threadDataService', [
   };
 
   var _fetchMessages = function(threadId) {
-    $http.get(_THREAD_HANDLER_PREFIX + threadId).success(function(data) {
+    $http.get(_THREAD_HANDLER_PREFIX 
+        + threadId.split(".")[1]).success(function(data) {
```

Comment:
Yes that does need to happen, but it may be out of scope of this commit (at least the js part). I can do that for all backend files in this commit. But for frontend js, I'd rather do it as a fresh commit because it will also involve testing those changes. Let me know if you feel otherwise.